app/Models/Language.php

```
@@ -417,6 +420,7 @@ public function delete(string $code): bool
      */
     public function setDefault(string $code): bool
     {
+        \App\Support\QueryCache::delete('i18n_languages');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\QueryCache' in method 'setDefault'. (undefined)

(StaticAccess)